lib/core/test/unit/Cardano/Wallet/DBSpec.hs

```diff
@@ -182,7 +186,12 @@ once xs = forM (Map.toList (Map.fromList xs))
 once_ :: (Ord k, Monad m) => [(k,v)] -> ((k,v) -> m a) -> m ()
 once_ xs = void . once xs
 
-type TxHistory = Map (Hash "Tx") (Tx, TxMeta)
+-- | Shorthand for the readTxHistory result type.
+type TxHistory = [(Hash "Tx", (Tx, TxMeta))]
+
+-- | Apply the default sort order (descending on time) to a 'TxHistory'.
+sortTxHistory :: TxHistory -> TxHistory
+sortTxHistory = reverse . L.sortOn (slotId . snd . snd)
```

Comment:
Pushed a small fix to satisty failing hlint preferring `Data.Ord.Down` here 2119408cc647327dde5c2786162ca3f7fdb25be2

Reply:
Thanks